src/transaction/xdc.ts

```diff
@@ -0,0 +1,811 @@
+ * @param body content of the transaction to broadcast
+ * @param provider url of the XDC Server to connect to. If not set, default public server will be used.
+ * @returns transaction id of the transaction in the blockchain
+ */
+export const sendXdcMintMultipleCashbackErc721Transaction = async (body: EthMintMultipleErc721, provider?: string) =>
+    xdcBroadcast(await prepareXdcMintMultipleCashbackErc721SignedTransaction(body, provider), body.signatureId);
+
+/**
+ * Send XDC ERC721 mint multiple transaction to the blockchain. This method broadcasts signed transaction to the blockchain.
+ * This operation is irreversible.
+ * @param body content of the transaction to broadcast
+ * @param provider url of the XDC Server to connect to. If not set, default public server will be used.
+ * @returns transaction id of the transaction in the blockchain
+ */
+export const sendXdcMintMultipleErc721Transaction = async (body: EthMintMultipleErc721, provider?: string) =>
+    xdcBroadcast(await prepareXdcMintMultipleErc721SignedTransaction(body, provider), body.signatureId);
+
+/**
+ * Send XDC ERC721 burn transaction to the blockchain. This method broadcasts signed transaction to the blockchain.
+ * This operation is irreversible.
+ * @param body content of the transaction to broadcast
+ * @param provider url of the XDC Server to connect to. If not set, default public server will be used.
+ * @returns transaction id of the transaction in the blockchain
+ */
+export const sendXdcBurnErc721Transaction = async (body: EthBurnErc721, provider?: string) =>
+    xdcBroadcast(await prepareXdcBurnErc721SignedTransaction(body, provider), body.signatureId);
+
+export const sendXdcUpdateCashbackForAuthorErc721Transaction = async (body: UpdateCashbackErc721, provider?: string) =>
+    xdcBroadcast(await prepareXdcUpdateCashbackForAuthorErc721SignedTransaction(body, provider), body.signatureId);
+
+/**
+ * Send XDC ERC721 transaction to the blockchain. This method broadcasts signed transaction to the blockchain.
+ * This operation is irreversible.
+ * @param body content of the transaction to broadcast
+ * @param provider url of the XDC Server to connect to. If not set, default public server will be used.
+ * @returns transaction id of the transaction in the blockchain
+ */
+export const sendXdcErc721Transaction = async (body: EthTransferErc721, provider?: string) =>
+    xdcBroadcast(await prepareXdcTransferErc721SignedTransaction(body, provider), body.signatureId);
+
+/**
+ * Send XDC ERC721 deploy to the blockchain. This method broadcasts signed transaction to the blockchain.
+ * This operation is irreversible.
+ * @param body content of the transaction to broadcast
+ * @param provider url of the XDC Server to connect to. If not set, default public server will be used.
+ * @returns transaction id of the transaction in the blockchain
+ */
+export const sendXdcDeployErc721Transaction = async (body: EthDeployErc721, provider?: string) =>
```

Comment:
ERC-1155 is not supported here, recent work from @Rachit-gaur . Please make a TODO here that after release, we should do the abstract methods for those operations and add support for 1155 for XCZ as well